patches/minecraft/net/minecraft/entity/player/EntityPlayer.java.patch

```diff
@@ -323,12 +326,21 @@
  
              if (blockpos == null)
              {
+@@ -1429,7 +1518,7 @@
+             }
+ 
+             this.func_70107_b((double)((float)blockpos.func_177958_n() + 0.5F), (double)((float)blockpos.func_177956_o() + 0.1F), (double)((float)blockpos.func_177952_p() + 0.5F));
+-        }
++        } else { p_70999_3_ = false; }
```

Comment:
Yeah. Patches have to be as small (and simple) as possible, so it's easier to apply them to newer versions.